packages/eslint-config-javascript/index.js

```diff
@@ -424,6 +429,8 @@ module.exports = {
         */
         'prefer-regex-literals': 'error',
 
+        'require-await': 'error',
```

Comment:
We are pretty consistent in this file with giving a reason in comment. Maybe (from the eslint docs directly): Asynchronous functions that don’t use await might not need to be asynchronous functions and could be the unintentional result of refactoring.